addons/binding/org.openhab.binding.smartthings/src/main/java/org/openhab/binding/smartthings/internal/converter/SmartthingsDefaultConverter.java

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright (c) 2010-2019 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.smartthings.internal.converter;
+
+import org.eclipse.smarthome.core.thing.ChannelUID;
+import org.eclipse.smarthome.core.thing.Thing;
+import org.eclipse.smarthome.core.types.Command;
+import org.eclipse.smarthome.core.types.State;
+import org.openhab.binding.smartthings.internal.SmartthingsStateData;
+
+/**
+ * Base converter class.
+ * The converter classes are responsible for converting "state" messages from the smartthings hub into openHAB States.
+ * And, converting handler.handleCommand() into messages to be sent to smartthings
+ *
+ * @author Bob Raker - Initial contribution
+ *
+ */
+public class SmartthingsDefaultConverter extends SmartthingsConverter {
```

Comment:
I don't understand the purpose of this class?! Just use `SmartthingsConverter` as a non abstract class.